yt/fields/field_detector.py

```diff
@@ -146,24 +147,17 @@ def __missing__(self, item):
                     self[item] = vv.ravel()
                 return self[item]
         elif finfo is not None and finfo.sampling_type == "particle":
-            if (
-                "particle_position" in (item, item[1])
```

Comment:
Is it ok that we're not checking `particle_position`, `particle_velocity`, or `particle_magnetic_field` anymore with these updates (unless I'm missing something?)?

Reply:
tbh I don't know. However, no test fails and I couldn't identify the place where it was used (if it was used).

Reply:
I digged a little. Normally those fields are aliases to some frontend specifc fields (like `particle_position` -> `Coordinates` for Gadget etc.). They'd only play a role if a frontend had a native field named like that, but then it should be defining it in its specific `IOHandler._vector_fields`.